service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/CreateAzureDatabaseStep.java

```diff
@@ -105,7 +105,7 @@ public StepResult undoStep(FlightContext context) throws InterruptedException {
     try {
       logger.info(
           "Attempting to delete database {} in server {} of resource group {}",
-          getResourceName(databaseResource),
```

Comment:
Why does getResourceName not work in this case?

Reply:
because it is a name of postgres instance.